tests/pyfunc/test_dependencies_functions.py

```diff
@@ -110,3 +113,100 @@ def mock_get_installed_version_fn(package, module=None):
                     "detecting model dependency mismatches"
                 )
             )
+
+def test_get_model_dependencies_read_req_file(tmp_path):
+    req_file = tmp_path / "requirements.txt"
+    req_file_content = """
+mlflow
+cloudpickle==2.0.0
+scikit-learn==1.0.2"""
+    req_file.write_text(req_file_content)
+
+    model_path = str(tmp_path)
+
+    # Test getting pip dependencies
+    assert Path(get_model_dependencies(model_path, format="pip")).read_text() == req_file_content
+
+    # Test getting pip dependencies will print instructions on databricks
+    with mock.patch("mlflow.pyfunc._logger.info") as mock_log_info:
+        get_model_dependencies(model_path, format="pip")
+        mock_log_info.assert_not_called()
+
+        mock_log_info.reset_mock()
+        with mock.patch("mlflow.pyfunc.is_in_databricks_runtime", return_value=True):
+            get_model_dependencies(model_path, format="pip")
+            mock_log_info.assert_called_once_with(
+                "To install these model dependencies in your Databricks notebook, run the "
+                f"following command: '%pip install -r {str(req_file)}'."
```

Comment:
```suggestion
                f"following command: '%pip install -r {req_file}'."
```

`str` can be removed.